### PR TITLE
Automatically connect using env. variables

### PR DESCRIPTION
**Pull request by thani-sh**

### CODE REVIEW

File: lib/apm.js [hunk 1]
```diff
@@ -170,3 +170,13 @@ Apm._getInfo = function(currentFiber) {
...
 };
+
+Apm.autoconfig = function () {
+  var appId = process.env.KADIRA_APP_ID || Meteor.settings.KADIRA_APP_ID;
+  var secret = process.env.KADIRA_APP_SECRET || Meteor.settings.KADIRA_APP_SECRET;
+  if(appId && secret) {
+    Apm.connect(appId, secret);
+  };
+}
+
```
Comment (arunoda) [inline lib/apm.js]: Create this in a separate file machan.

**Comment by arunoda** [comment]
- [x] Add support for Meteor.settings as well
- [ ] Prevent connecting multiple times.

**Comment by thani-sh** [comment]
Done

File: autoconfig.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+Meteor.startup(function () {
+  var appId = process.env.KADIRA_APP_ID || Meteor.settings.KADIRA_APP_ID;
+  var secret = process.env.KADIRA_APP_SECRET || Meteor.settings.KADIRA_APP_SECRET;
+  if(appId && secret) {
+    Apm.connect(appId, secret);
+    Apm.connect = function () {
+      throw new Error('APM: Already connecting using environment variables');
```
Comment (arunoda) [inline autoconfig.js]: We need to detect if this is connected with Meteor Settings or Env Variables.

File: autoconfig.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+Meteor.startup(function () {
+  console.warn('APM: Trying to connect');
+  if(process.env.KADIRA_APP_ID && process.env.KADIRA_APP_SECRET) {
+    console.warn('APM: connected using environment variables');
+    Apm.connect(process.env.KADIRA_APP_ID, process.env.KADIRA_APP_SECRET);
+    Apm.connect = function () {
+      throw new Error('APM: already connected using environment variables');
+    }
+  } else if(Meteor.settings.KADIRA_APP_ID && Meteor.settings.KADIRA_APP_SECRET) {
```
Comment (arunoda) [inline autoconfig.js:10]: This needs to be `Meteor.settings.kadira.appId`. User will expose something like this.

``` js
{
  "kadira": {
    "appId": "kdkdsfjk32",
    "appSecret": "jdhwdfjk3r2kr"
  }
}
```

**Comment by arunoda** [comment]
This is already added on the kadira package.

**Pull request closed without merging**